core/sodasql/common/config_helper.py

```diff
@@ -0,0 +1,121 @@
+#  Copyright 2021 Soda
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#   http://www.apache.org/licenses/LICENSE-2.0
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+
+import logging
+from typing import Dict, Optional
+import uuid
+import yaml
+
+from sodasql.scan.file_system import FileSystemSingleton
+
+logger = logging.getLogger(__name__)
+
+class ConfigHelper:
+    """Helper class for handling global Soda config.
+    """
+    DEFAULT_CONFIG = {
+        'skip_telemetry': False,
+        'user_cookie_id': str(uuid.uuid4())
+    }
+    LOAD_PATHS = ["~/.soda/config.yml", ".soda/config.yml"]
+    __instance = None
+    __config: Dict = None
+    file_system = FileSystemSingleton.INSTANCE
+
+    @staticmethod
+    def get_instance(path: Optional[str] = None):
+        if ConfigHelper.__instance == None:
```

Comment:
I think the preferred comparison operator is `is None` here

Reply:
I think flake8 would surface this. Might be a good idea to set it up as a pre-commit hook